cmd/traefik/traefik.go

```diff
@@ -256,3 +259,41 @@ func checkNewVersion() {
 		}
 	})
 }
+
+func stats(globalConfiguration *configuration.GlobalConfiguration) {
+	if globalConfiguration.SendAnonymousUsage {
+		log.Info(`
+Stats collection is enabled.
+Many thanks to contribute to Traefik improvement by allowing us to receive anonymous information from your configuration.
```

Comment:
I think it must say (**emphasis** by me)

_Many thanks **for contributing** to **Traefik's** improvement [...]_